docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/msg/motion_factor.md

```diff
@@ -0,0 +1,37 @@
+# autoware_ad_api_msgs/msg/MotionFactor
+
+## Definition
+
+```txt
+# constants for reason
+uint16 INTERSECTION=1
+uint16 MERGE_FROM_PRIVATE_ROAD=2
+uint16 CROSSWALK=3
+uint16 WALKWAY=4
+uint16 STOP_LINE=5
+uint16 DETECTION_AREA=6
+uint16 NO_STOPPING_AREA=7
```

Comment:
What is `NO_STOPPING_AREA` and how does the vehicle behaves in this case? Stopping before the no_stopping_area? :thinking:

Reply:
I think the meaning is "prohibited to stop area". 
https://www.think-sp.com/2012/11/02/tw-teishikinshi-bubun/

Reply:
It seems there are three similar types (parking/standing/stopping)?
http://kenkitagawa.cocolog-nifty.com/blog/2013/03/no-standing-887.html

Reply:
> It seems there are three similar types (parking/standing/stopping) and stopping is correct?

I think so.